gnosis/safe-react
# Tx decoded data: fix addresses array

**What**:
Closes #1758

**Why**:
There was a problem when tx data was an array of strings. All the addresses were shown as a single address, breaking the copy and open explorer button. 

**How**:
I added a new handler for when the type is `address[]`.

**How to test it**:
1. Go to https://gnosis-safe.io/app/#/safes/0xfeb4acf3df3cdea7399794d0869ef76a6efaff52/transactions
2. Open tx with nonce 216
3. Check decoded data
